hack/verify-codegen.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/sh
 
 if [ "$IS_CONTAINER" != "" ]; then
-  set -x
+  set -xe
   go generate ./pkg/types/installconfig.go
-  go generate ./pkg/rhcos/ami.go
-  set +x
+  # See https://github.com/openshift/installer/pull/5447#discussion_r762340594
+  # go generate ./pkg/rhcos/ami.go
```

Comment:
Fixed with https://github.com/openshift/installer/pull/5466.

Reply:
sounds like i should just delete those two commented-out lines now then @staebler?

Reply:
No, leave them commented out. This PR will likely merge before mine.